utils/src/main/kotlin/DeclaredLicenseProcessor.kt

```diff
@@ -76,4 +76,13 @@ data class ProcessedDeclaredLicense(
             unmapped = emptyList()
         )
     }
+
+    /**
+     * Get the list of all mapped and unmapped licenses.
+     */
+    fun getAllLicenses(): List<String> {
+        val licenses = unmapped.toMutableList()
+        spdxExpression?.let { licenses.addAll(0, it.licenses()) }
+        return licenses
```

Comment:
You could make it a one-liner: `spdxExpression?.licenses().orEmpty() + unmapped`

Reply:
Good Idea!